geopandas/io/file.py

```diff
@@ -27,6 +27,10 @@ def read_file(filename, **kwargs):
             f_filt = f
         gdf = GeoDataFrame.from_features(f_filt, crs=crs)
 
+        # re-order with column order from metadata, with geometry first
+        columns = ["geometry"] + list(f.meta["schema"]["properties"])
```

Comment:
Just tested this myself, and it seems that the resulting geometry column is always called 'geometry'. So in that case this code is fine.

Regarding the first/last position of the geometry column, do you know if there are any precedents? (other io functions that do this in a certain way?)